frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingsTableConfig.tsx

```diff
@@ -98,21 +98,24 @@ const tableHeaders: IDataColumn[] = [
   },
 ];
 
-const makeWindowsRows = ({ os_settings }: IHostMdmData) => {
-  if (
-    !os_settings?.disk_encryption?.status ||
-    !isWindowsDiskEncryptionStatus(os_settings.disk_encryption.status)
-  ) {
+const makeWindowsRows = ({ profiles, os_settings }: IHostMdmData) => {
+  if (!profiles) {
```

Comment:
Didn't catch this the first time, but I think we still want to return the BitLocker row if there are no profiles, right?

Reply:
ohhh, awesome catch, thanks! I pushed something. I don't know how important it is to return `null` instead of an empty array, but I added extra logic to keep the existing behavior.